pkg/registry/extensions/podsecuritypolicy/storage/storage_test.go

```diff
@@ -20,9 +20,9 @@ import (
 	"testing"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
are you planning to move the `pkg/registry/extensions/podsecuritypolicy` package itself as a followup?

Reply:
Yes, as its listed in the plan and just to make changes granular, small and easy to review.